src/miner.cpp

```diff
@@ -234,8 +234,9 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript &sc
         nLockTimeCutoff =
             (STANDARD_LOCKTIME_VERIFY_FLAGS & LOCKTIME_MEDIAN_TIME_PAST) ? nMedianTimePast : pblock->GetBlockTime();
 
-        addPriorityTxs(pblocktemplate.get());
-        addScoreTxs(pblocktemplate.get());
+        std::vector<const CTxMemPoolEntry*> txs;
+        addPriorityTxs(&txs);
```

Comment:
need to be run though formatting...

Also, txs could be changed maybe to vtxe...i think we've used txe to described  CTxMempool entries before and which also differentiates them from simple transactions.

Other than that it looks good to me...